### ISSUE TITLE
Opal.use_gem throw exception in 0.9

### ISSUE THREAD

**Issue by fazibear** [opal/opal#1175]
`Opal.use_gem` throws exception in Opal `0.9-beta1`
```
/Users/fazibear/.rubies/2.2.3/lib/ruby/gems/2.2.0/gems/opal-0.9.0.beta1/lib/opal/paths.rb:36:in `require_paths_for_gem': undefined method `runtime_dependencies' for nil:NilClass (NoMethodError)
    from /Users/fazibear/.rubies/2.2.3/lib/ruby/gems/2.2.0/gems/opal-0.9.0.beta1/lib/opal/paths.rb:37:in `block in require_paths_for_gem'
    from /Users/fazibear/.rubies/2.2.3/lib/ruby/gems/2.2.0/gems/opal-0.9.0.beta1/lib/opal/paths.rb:36:in `each'
    from /Users/fazibear/.rubies/2.2.3/lib/ruby/gems/2.2.0/gems/opal-0.9.0.beta1/lib/opal/paths.rb:36:in `require_paths_for_gem'
    from /Users/fazibear/.rubies/2.2.3/lib/ruby/gems/2.2.0/gems/opal-0.9.0.beta1/lib/opal/paths.rb:27:in `use_gem'
```

**Comment by elia** [comment]
@fazibear how did you get that error? can you tell me the rubygems version? (`gem -v`, seems fine here with 2.4.8)

**Comment by elia** [comment]
Anyway I added some specs for it with 66a2a8da726b7c6a52eede5eae576f82c799a923

**Comment by fazibear** [comment]
Also have rubygems `2.4.8`. gemspec points to beta `s.add_dependency 'opal', '~> 0.9beta'`.
Checked with master and it works fine. So it was fixed after `0.9.0.beta1` was released.

**Comment by elia** [comment]
Going to close this, and releasing beta2
